content/blog/2023/03/03/2023-03-03-miniJen-is-alive.adoc

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title: "miniJen is alive!"
+tags:
+- jenkins
+- miniJen
+- riscv
+- aarch64
+- arm32
+- fosdem
+authors:
+- gounthar
+opengraph:
+image: /images/post-images/2023/03/03/2023-03-03-miniJen-is-alive/image2.png
+discourse: true
+---
+
+= miniJen
+
+The Jenkins multi-architecture CPU instance
+
+image::/images/post-images/2023/03/03/2023-03-03-miniJen-is-alive/fosdem_2023_booth_display.jpg[miniJen as a FOSDEM display on the booth,839]
+
+== What is that contraption?
+
+Nope, it's not a robot of some sort, it won't move by itself.
+It's not https://en.wikipedia.org/wiki/Cerebro[Cerebro] from Professor Xavier; no, it can't https://all3dp.com/2/raspbery-pi-drone-simply-explained/[fly] either.
+What you're looking at is a Jenkins "`https://www.jenkins.io/doc/#what-is-jenkins[instance]`".
+It is composed of a https://www.jenkins.io/doc/book/using/using-agents/#using-jenkins-agents[controller] (the "`brain`" or conductor) and three https://www.jenkins.io/doc/book/using/using-agents/[agents] (the workers or musicians if we continue a little further with the metaphor).
```

Comment:
Use site local links so that the links work when building the development site.
```suggestion
It is composed of a link:/doc/book/using/using-agents/#using-jenkins-agents[controller] (the "`brain`" or conductor) and three link:/doc/book/using/using-agents/[agents] (the workers or musicians if we continue a little further with the metaphor).
```